pkg/operator/edgefs/cluster/target/pod.go

```
@@ -639,8 +639,12 @@ func (c *Cluster) makeStatefulSet(replicas int32, rookImage string, dro edgefsv1
 	k8sutil.SetOwnerRef(&statefulSet.ObjectMeta, &c.ownerRef)
 
 	if c.NetworkSpec.IsMultus() {
-		k8sutil.ApplyMultus(c.NetworkSpec, &statefulSet.ObjectMeta)
-		k8sutil.ApplyMultus(c.NetworkSpec, &statefulSet.Spec.Template.ObjectMeta)
+		if err := k8sutil.ApplyMultus(c.NetworkSpec, &statefulSet.ObjectMeta); err != nil {
+			logger.Errorf("failed to apply multus spec to stateful set metadata. %v", err)
```

let's return the errors here